docs/kh3/battlequotes.md

```diff
@@ -245,3 +246,38 @@ Their type is SQEXSEADSoundBank or SQEXSEADSound.
 | 029       | \<Attack Grunt\> | |
 
 [To top](#kingdom-hearts-iii-battle-quotes)
+
+### Xemnas
+
+`/Game/Sound/VOICE/Localization/en/battle/ENEMY/Battle_Xemnas.uasset`
+
+| Audio Nr. | Quote                            | Note         |
+|:---------:| -------------------------------- | ------------ |
+| 000       | \<strike sounds\>                |              |
+| 001       | "To the Void"                    |              |
+| 002       | "No More"                        |              |
+| 003       | "Erase"                          |              |
+| 004       | "Burst"                          |              |
+| 005       | "There"                          |              |
+| 006       | "Strike"                         |              |
+| 007       | "Anger and Hatred reign supreme" |              |
+| 008       | "Feel the Emptyness"             |              |
+| 009       | "Insolence"                      |              |
+| 010       | \<hurt sounds\>                  | Damage taken |
+| 011       | \<hurt sounds\>                  | Damage taken |
+| 012       | \<hurt sounds\>                  | Damage taken |
+| 013       | \<hurt sounds\>                  | Damage taken |
+| 014       | \<hurt sounds\>                  | Damage taken |
+| 015       | \<hurt sounds\>                  | Damage taken |
+| 016       | "Are you Ready"                  |              |
+| 017       | "Grovel"                         |              |
+| 018       | "Pay Attention"                  |              |
+| 019       | "Begone"                         |              |
+| 020       | \<strike sounds\>                |              |
+| 021       | \<strike sounds\>                |              |
+| 022       |                                  | failed to extract |
```

Comment:
For line 22, could we change the comment to "?" instead? If only for the sake of being completely clear that Xemnas does not, in fact, say "failed to extract". 😆 
Otherwise, thanks for the addition. 👍